test/e2e/monitoring_stack_controller_test.go

```diff
@@ -854,6 +872,7 @@ func namespaceSelectorTest(t *testing.T) {
 
 	stopChan := make(chan struct{})
 	defer close(stopChan)
+	//nolint
 	if pollErr := wait.Poll(5*time.Second, 2*time.Minute, func() (bool, error) {
```

Comment:
Change line 876 for => `if pollErr := wait.Poll(15*time.Second, 5*time.Minute, func() (bool, error) {`

Reply:
done, let's see